### ISSUE TITLE
Our coverage is below 90%

### ISSUE THREAD

**Issue by lemire** [RoaringBitmap/roaring#57]
More testing is needed:

https://gocover.io/github.com/RoaringBitmap/roaring

**Comment by cornelk** [comment]
The URL gives an 404 error.

**Comment by lemire** [comment]
Looks like gocover.io went under?

**Comment by cornelk** [comment]
https://coveralls.io/ looks like a good alternative, free for open source projects.

**Comment by lemire** [comment]
I use coveralls.io in other non-Go projects, but I do not know how to use it with Go. Do you?

**Comment by cornelk** [comment]
No, I never used it. I saw another Go project using it: https://coveralls.io/github/go-gorp/gorp

**Comment by lemire** [comment]
I know that some projects use it, but I tried to follow the fuzzy instructions and it doesn't work. There are some mysterious steps involved. Looks like magic.

**Comment by lemire** [comment]
https://coveralls.io/github/RoaringBitmap/roaring

**Comment by cornelk** [comment]
I added the coverage badge to the README, the website still does not show content for this project: "There have been no builds for this repo."

**Comment by lemire** [comment]
@cornelk 

I am aware that it does not yet work. I tried something else this morning. Let us see.

**Comment by lemire** [comment]
Coverage badge should be ok.

**Comment by cornelk** [comment]
Great, I started working on some tests to test the panic edge cases now.
